src/pages/auth/components/notification.js

```diff
@@ -0,0 +1,21 @@
+import 'react-toastify/dist/ReactToastify.css'
+import { toast } from 'react-toastify'
+import { en } from '../../../localization/strings/en'
+
+export const loginError = () => {
+    toast.error(en['Wrong email or password!'], {
+    position: toast.POSITION.TOP_LEFT,
+  })
+}
+
+export const registerError = () => {
+    toast.error(en['Ooops! Try again'], {
+    position: toast.POSITION.TOP_LEFT,
+  })
+}
+
+export const registerSuccess = () => {
+  toast.success(en['We send a verify letter to your email!'], {
```

Comment:
Btw translations will not works in this way, you need to use sentenses from {t} = useTransltions()


But you will need to move text with parameters

Reply:
done